src/wiki/getting-started/data-location.md

```diff
@@ -0,0 +1,27 @@
+---
+eleventyNavigation:
+  key: Data storage
+  parent: Getting Started
+  order: 11
+--- 
+# Data Location
+
+Prism Launcher stores your data in your OS's standard application data folder. Below is a table containing the default locations:
+
+| OS               | Folder                                                          |
+| ---------------- | --------------------------------------------------------------- |
+| Windows          | `%APPDATA%/PrismLauncher`                                       |
+| macOS            | `~/Library/Application Support/PrismLauncher`                   |
```

Comment:
sometimes prism can use /PolyMC instead of /PrismLauncher. document that

Reply:
Looked at the launcher code and we don't do that anymore(if it was ever a thing).
The worst we do is fall back to "polymc.cfg" or  "multimc.cfg"   if  "prismlauncher.cfg" doesn't exists.